docs/assets/js/main.js

```diff
@@ -23,12 +22,12 @@ function toggleHeaderMenu() {
 }
 
 function navigateBack() {
-    if (window.location.pathname.includes('/request-money/')) {
-        window.location.href = '/hubs/request-money';
-    } else if (window.location.pathname.includes('/send-money/')) {
-        window.location.href = '/hubs/send-money';
+    const hubs = JSON.parse(document.getElementById('hubs-data').value);
+    const hubToNavigate = hubs.find(hub => window.location.pathname.includes(hub));
```

Comment:
@marcochavezf looks like this is failing lint

Reply:
It was failing because of this eslint error: `Prefer '_.find' over the native function`, but I'm disabling it because we're not using underscore yet in ExpensifyHelp